### PR TITLE
feat: Add notifications for issues.

### PR DESCRIPTION
**Pull request by kartik-gupta-ij**
# UI for https://github.com/qdrant/qdrant/issues/3471

## Layout section 
- [x] Notification for error(issues) with count.
 
## Modal  to view issues.
- [x]  Layout of modal 
- [x] Action button

### CODE REVIEW

File: src/pages/Home.jsx [hunk 1]
```diff
@@ -59,6 +63,24 @@ export default function MiniDrawer() {
...
+      setIssusesCount(issues.data.result.issues.length);
+    } catch (error) {
+      console.log('error', error);
+    }
+  }
+
+  React.useEffect(() => {
+    const interval = setInterval(() => {
+      getIssuesCount();
+      console.log('This will run every 5 seconds!', issuses);
```
Comment (generall) [inline src/pages/Home.jsx:79]: I don't think we need it every 5 seconds. Let's maybe keep it to only once on the page load

**Review by kartik-gupta-ij — COMMENTED** [review]
https://github.com/qdrant/qdrant-web-ui/assets/84975264/b6ffa346-6b5a-4cb4-90df-c4286b9efca5

@generall, @coszio Please review!

**Comment by timvisee** [comment]
@kartik-gupta-ij That looks fantastic!

I do think I prefer a overlaying sidebar, just like the green commands button on the bottom left. It has more room for issues, and keeps the UI consistent. Wdyt?

And we should probably render the request result a bit better, but I don't have a direct suggestion there.

**Review by coszio — COMMENTED** [review]
I love seeing the actual request in the button to solve.

However, I agree with @timvisee. Creating an index is an important choice, so it would be more fitting to have a dedicated page for this.

For the content of the notification. The title should be more friendly. This can be extracted from the ID, up to the first `/` (slash). Otherwise I can include this as an additional field in `GET /issues`

**Review by kartik-gupta-ij — COMMENTED** [review]
https://github.com/qdrant/qdrant-web-ui/assets/84975264/02cfc3d9-79a6-4a0b-b7b8-044cd01f6f5f

Feedbacks @timvisee , @coszio ?

**Comment by kartik-gupta-ij** [comment]
> Screen.Recording.2024-05-23.at.8.02.01.PM.mov 
> Feedbacks @timvisee , @coszio ?

Uploading Screen Recording 2024-05-23 at 8.02.01 PM.mov…

Minor updates!

**Comment by coszio** [comment]
The displaying of issues is looking better 👍 

Another reason for having a dedicated page is that sometimes it is helpful to clear the issues with a button that calls `DELETE /issues`. Use-case: just playing around / developing with filters, so that using a wrong key name will trigger the issue for it, and it doesn't mean that an index should be created.

**Review by generall — APPROVED** [review]

**Pull request merged**